docker-entrypoint.sh

```diff
@@ -6,8 +6,6 @@ IFS=$'\n\t'
 
 export S3_ACL=${S3_ACL:-private}
 
-test $MOUNT_POINT
-rm -rf ${MOUNT_POINT}
```

Comment:
@rjocoleman I get an error on first run with original version of docker-compose.sh. Actually my PR is just about Dockerfile, docker-entrypoint.sh is a quickfix